### ISSUE TITLE
[Bug?] Scolled event is only triggered when swiping manually

### ISSUE THREAD

**Issue by Symxn** [alexrainman/CarouselView#578]
Hey, i like your control but I have a little note:

The scrolled event is only triggered if the user swipes back and forth manually. But not when the user stops swiping and the animation returns the element to the correct position.
Can you please change that? :)

**Comment by chaoyebugao** [comment]
Yes. If you scroll to 50% (center screen，just an example data) and lift finger,  then the view still scroll but the scrolled event won't trigger, some how I won't able to know the view's actual scroll state.
Only in Android by the way.
